.github/workflows/flutter_beta.yml

```diff
@@ -25,6 +25,7 @@ jobs:
       with:
         channel: ${{ env.FLUTTER_CHANNEL }}
         cache: true
+    - uses: bluefireteam/melos-action@v3
     - run: flutter pub get
```

Comment:
I think the pub get here is redundant now since melos does this during bootstrap.
Same in the other jobs.

Reply:
Fixed